packages/mux-player/test/web-test-runner.config.mjs

```diff
@@ -24,7 +24,7 @@ const config = {
     report: true,
     include: ['src/**/*'],
   },
-  testsFinishTimeout: 300000,
+  testsFinishTimeout: 600000,
```

Comment:
NOTE: bumped this on the hopes that it would resolve resource issues. It appears that the tests are generally slowing down based on available resources (e.g. memory) on GHA env.